hostinfo/hostinfo.go

```diff
@@ -173,8 +174,19 @@ var (
 	firewallMode      atomic.Value // of string
 )
 
-// SetDeviceModel sets the device model for use in Hostinfo updates.
-func SetDeviceModel(model string) { deviceModelAtomic.Store(model) }
```

Comment:
Sorry, I just noticed this is being deleted but we use this from our iOS/macOS app.

Can you keep this part still?

Reply:
No biggie. Added it back!